cli/flox/src/commands/mod.rs

```diff
@@ -847,188 +908,162 @@ enum LocalDevelopmentCommands {
+        When no arguments are specified,
+        all packages in the environment are upgraded if possible.
+        A package is upgraded if its version, build configuration,
+        or dependency graph changes.\n\n
+
+        Packages to upgrade can be specified by group name.
+        Packages without a specified pkg-group in the manifest
+        are placed in a group named 'toplevel'.
+        The packages in that group can be upgraded without updating any other groups
+        by passing 'toplevel' as the group name.\n\n
+
+        A single package can only be specified to upgrade by ID
+        if it is not in a group with any other packages.
+    "}))]
+    Upgrade(#[bpaf(external(upgrade::upgrade))] upgrade::Upgrade),
+
+    /// Uninstall installed packages from an environment
+    #[bpaf(
+        command,
+        long("remove"),
+        long("rm"),
+        footer("Run 'man flox-uninstall' for more details.")
+    )]
+    Uninstall(#[bpaf(external(uninstall::uninstall))] uninstall::Uninstall),
 }
 
-impl LocalDevelopmentCommands {
-    async fn handle(self, config: Config, flox: Flox) -> Result<()> {
+impl ModifyCommands {
+    async fn handle(self, flox: Flox) -> Result<()> {
         match self {
-            LocalDevelopmentCommands::Init(args) => args.handle(flox).await?,
-            LocalDevelopmentCommands::Activate(args) => args.handle(config, flox).await?,
-            LocalDevelopmentCommands::Edit(args) => args.handle(flox).await?,
-            LocalDevelopmentCommands::Install(args) => args.handle(flox).await?,
-            LocalDevelopmentCommands::Uninstall(args) => args.handle(flox).await?,
-            LocalDevelopmentCommands::List(args) => args.handle(flox).await?,
-            LocalDevelopmentCommands::Search(args) => args.handle(config, flox).await?,
-            LocalDevelopmentCommands::Show(args) => args.handle(flox).await?,
-            LocalDevelopmentCommands::Delete(args) => args.handle(flox).await?,
-            LocalDevelopmentCommands::Services(args) => args.handle(config, flox).await?,
+            ModifyCommands::Install(args) => args.handle(flox).await?,
+            ModifyCommands::List(args) => args.handle(flox).await?,
+            ModifyCommands::Edit(args) => args.handle(flox).await?,
+            ModifyCommands::Include(args) => args.handle(flox).await?,
+            ModifyCommands::Upgrade(args) => args.handle(flox).await?,
+            ModifyCommands::Uninstall(args) => args.handle(flox).await?,
         }
         Ok(())
     }
 }
 
-/// Sharing Commands
+/// Share with others
 #[derive(Bpaf, Clone)]
-enum SharingCommands {
+enum ShareCommands {
+    /// Build packages for Flox
+    #[bpaf(
+        command,
+        // TODO: NEF
+        header(
+            "Build packages from the manifest's 'build' table, or run clean subcommand if specified."
+        ),
+        footer("Run 'man flox-build' for more details.")
+    )]
+    Build(#[bpaf(external(build::build))] build::Build),
```

Comment:
`build` is a bit out of place IMO, but i agree without creating a new group for building/package development this is probably closest.
Since build is also itself a group, alternatively build could be standing on its own as well.